### ISSUE TITLE
The Weed tab of Sprout Import is broken when using a Postgres database

### ISSUE THREAD

**Issue by BenParizek** [barrelstrength/craft-sprout-import#21]
The Weed tab of Sprout Import is broken when using a Postgres database on Craft CMS 3.0.0-RC17.1 (PHP 7.1.15, PostgreSQL 10.2):

```
Internal Server Error
SQLSTATE[42883]: Undefined function: 7 ERROR: function group_concat(integer) does not exist LINE 1: SELECT GROUP_CONCAT(id) ids, "type", "details", COUNT(1) as ...

HINT: No function matches the given name and argument types. You might need to add explicit type casts. The SQL being executed was: SELECT GROUP_CONCAT(id) ids, "type", "details", COUNT(1) as total, "dateCreated" FROM "sproutimport_seeds" GROUP BY "dateCreated", "details", "type" ORDER BY "dateCreated" DESC
```

### Notify

- [ ] [15569857501](https://app.intercom.io/a/apps/dfa024eb83c11784227312bba407fb3d35e00fa7/inbox/inbox/714050/conversations/15569857501)

**Comment by pdaleramirez** [comment]
This bug is fixed on beta versions of sprout-import

**Comment by BenParizek** [comment]
Fixed in release `v1.0.0-beta.12`
